src/components/BookList.css

```diff
@@ -1,6 +1,8 @@
 .book-list {
   width: 70%;
   padding: 24px;
+  color: rgb(75, 170, 202);
+  font-family: 'Roboto Slab', serif;
```

Comment:
- [ ] @wacoo Great work styling your booklist page, however the fonts, text color, background color and width differ from the design given, check the comparison below for better understanding:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/wacoo/react-book-store/assets/61408860/15408b3a-7ae6-4bae-931f-751ecf155d0e)| ![image](https://github.com/wacoo/react-book-store/assets/61408860/53ec1645-87f6-4ba1-ba3f-dd492f41dda4)

Reply:
- [ ] @wacoo Great work styling our add book section however the font text color and button color used differ with the design given, check the comparison below:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/wacoo/react-book-store/assets/61408860/176766d7-2a79-4c1f-a4ac-83fc9bb80d68)  | ![image](https://github.com/wacoo/react-book-store/assets/61408860/8fe42f63-e81b-4c51-a2c7-7c3e3f7a7078)